experimental/InjectiveResolutions/src/InjectiveResolutions.jl

```diff
@@ -1048,18 +1023,18 @@ false
     if is_zero(E)
       d = -1
     else 
-      d = dim(E)
+      d = krull_dim(E)
     end
-    cod = dim(R_B) - d
+    cod = krull_dim(R_B) - d
     if cod < j+n+3
         return false
     end               # S2 condition not satisfied
   end
 
   Jac = ideal(R, minors(map_entries(R, jacobian_matrix(gens(modulus(R)))), n))  # Compute minors of the Jacobian
-  d = dim(Jac)                   # Get dimension of the Jacobian
+  d = krull_dim(Jac)                   # Get dimension of the Jacobian
   d < 0 && (d = -Inf)            # Handle negative dimensions
```

Comment:
I am not going to change this as I did not test it, and this is existing code, not the "fault" of this PR -- but I think this might be causing type instability, as `-Inf` is a `Float64`; really should be using `-inf` here.
```suggestion
  d < 0 && (d = -inf)            # Handle negative dimensions
```

@emikelsons please in your PR, try making this change.